src/main/scala/org/bitcoins/core/protocol/script/ScriptPubKey.scala

```diff
@@ -26,14 +29,21 @@ sealed trait ScriptPubKey extends NetworkElement with BitcoinSLogger {
    */
   lazy val asm : Seq[ScriptToken] = ScriptParser.fromBytes(bytes)
 
+  def p2shAddress(network : NetworkParameters) : P2SHAddress = {
```

Comment:
Also FWIW, decoding a scriptPubKey (of any type) in hex in the bitcoin daemon returns the P2SH address that it would generate if that scriptPubKey was put into a P2SHScriptPubKey.

Removing it for time being.